src/Sentry/SessionEndStatus.cs

```diff
@@ -13,10 +13,15 @@ public enum SessionEndStatus
     /// <summary>
     /// Session ended with an unhandled exception.
     /// </summary>
+    Unhandled,
+
+    /// <summary>
+    /// Session ended with a terminal unhandled exception.
+    /// </summary>
     Crashed,
 
     /// <summary>
     /// Session ended abnormally (e.g. device lost power).
     /// </summary>
-    Abnormal
+    Abnormal,
```

Comment:
Can/Would we ever set this in practice? I see it's used in the tests...

Reply:
That's a good point and I think we're abusing this in the Unity SDK. It's a bit outside of the scope of this PR but I'll check when taking a stab at properly shutting down sessions during application closure.